src/inclusion.md

```diff
@@ -148,8 +148,8 @@ We have provision for people with hearing, vision and mobility requirements.
   <h3>Dietary requirements</h3>
 
   <p>
-    We aim to meet all dietary requirements.
-    You will be able to specify these when purchasing your ticket.
+    Vegan, vegetarian, lactose and gluten-free options will be available on our lunch time menus.
```

Comment:
in that case, lactose also needs changing to lactose-free (I was just trying to avoid the repetition)